.travis.yml

```diff
@@ -6,3 +6,4 @@ install:
   - pip install -r test-requirements.txt
 script:
   - flake8 .
+  - python -m pytest test/test.py
```

Comment:
You can do `pytest tests/test_*.py`, so whenever we add new test files they will be automatically executed without the need to change this.

`pytest` should be added to `test-requirements.txt` too (if it's not already there).